packages/sirius-web/frontend/sirius-web-application/src/modals/upload-document/UploadDocumentModal.tsx

```diff
@@ -91,3 +103,110 @@ export const UploadDocumentModal = ({ editingContextId, onClose }: UploadDocumen
     </Dialog>
   );
 };
+
+export const UploadDocumentSplitButton = ({
+  readOnly,
+  onPermissionSelected,
+  disabled,
+  loading,
+}: UploadDocumentSplitButtonProps) => {
+  const [state, setState] = useState<UploadDocumentSplitButtonState>({
+    selected: false,
```

Comment:
Is there anyone here reading this selected value? What is the point of this?

Reply:
No, I should have removed it, it was part of a previous implementation where the list of actions wasn't bounded to the two options we have right now.